core/math/math_funcs.h

```diff
@@ -221,7 +221,10 @@ class Math {
 
 	static _ALWAYS_INLINE_ double abs(double g) { return absd(g); }
 	static _ALWAYS_INLINE_ float abs(float g) { return absf(g); }
-	static _ALWAYS_INLINE_ int abs(int g) { return g > 0 ? g : -g; }
+	static _ALWAYS_INLINE_ int8_t abs(int8_t g) { return g > 0 ? g : -g; }
+	static _ALWAYS_INLINE_ int16_t abs(int16_t g) { return g > 0 ? g : -g; }
+	static _ALWAYS_INLINE_ int32_t abs(int32_t g) { return ::abs(g); }
+	static _ALWAYS_INLINE_ int64_t abs(int64_t g) { return ::abs(g); }
```

Comment:
```suggestion
	static _ALWAYS_INLINE_ int64_t abs(int64_t g) { return ::labs(g); }
```

Careful, I'm not sure if `abs(long)` is guaranteed to exist: https://en.cppreference.com/w/c/numeric/math/abs

Reply:
Yeah a big part of why we even have `Math::abs` instead of relying on stdlib is that `abs` overloads are/used to be ambiguous and would cause errors with MSVC (probably VS 2013 back then).

Reply:
Well... https://github.com/godotengine/godot/actions/runs/13946462884/job/39034795168?pr=69406
I'll go back to original code.

Reply:
It might need to be `llabs(long long)` for `int64_t`. I really despise `short`/`int`/`long`/`long long` types :D

Reply:
Yeah `llabs` may work. I _think_ it's guaranteed to be `int64_t`? But yea, it would have been easier with C++ APIs rather than those old C headers...